### DISCUSSION TITLE
404 Error on First Project

### DISCUSSION THREAD

**Discussion by ashleycerku** [community/community#38096] [Other Feature Feedback, Questions, & Ideas]
Hi all,

I'm following an online tutorial to test out using 3D location-based images via a website, and I am stuck on the last bit. The tutorial says:
"One last element: we have to tell Github to serve this page up on the web. Go to the cogwheel ‘settings’ icon and scroll down to Github Pages. Under the drop down under ‘Source’ select ‘Master’. Github will build the html and make it available at a location (it’ll reload and say ‘your page is published at…')."

Although the directions are from an older version, I can sort of tell what to do and have a link to a html, but keep getting the 404 error when I try to open it. Thoughts? Much appreciated!

https://acmsu.github.io/TEST-2/ 

![image](https://user-images.githubusercontent.com/116527244/199565183-b1e18bbe-78de-4ea5-9afd-8d094c929569.png)

**Comment by david-engelmann** [comment]
You branch is currently set to main, In your description you said it should be set to "Master". Is that an option? Additionally, should the github page be built from / (root) or is there a subdirectory that it should actually be built from?

**Reply by ashleycerku** [reply]
Thanks for taking a look! These are the options I see:
![image](https://user-images.githubusercontent.com/116527244/199568502-45d6341b-83f0-4dac-afc6-001a97b8eadb.png)
![image](https://user-images.githubusercontent.com/116527244/199568853-1c2d0c02-6d7c-4adc-acd4-36921269bd8c.png)

This is the tutorial I am following if that helps: https://graddh.netlify.app/docs/tutorials/ar-vr/?s=09

**Comment by radar-blip** [comment]
You have an `index.html.nb`. What is the .nb for?
I think you need to rename that to just index.html

**Reply by ashleycerku** [reply]
That was what the tutorial said to do. So, I just did another one without the .nb and got this:
<img width="286" alt="image" src="https://user-images.githubusercontent.com/116527244/199620710-cd460f30-6e0d-4855-8365-a8e008d7626b.png">

**Reply by radar-blip** [reply]
Ok, turns out .nb is a legit file extension (for new notebooks)?
Anyway, I didnt read to much into it but I'm not so sure Github will let you use that. 
Github Pages are for static sites mostly, not interactive sites.

**Reply by ashleycerku** [reply]
Hmm, ok. Thanks for looking into it. The tutorial said to use GitHub so not sure then. Thanks for your help!

**Reply by radar-blip** [reply]
Yeah, I skimmed through that tutorial pretty fast, it did say something about GitHub but I'm not sure either. 
Are you trying to make an AR App? 
Anyway, keep trying, I'm no expert in that area.
Good luck!

**Comment by jfullstackdev** [comment]
it's already hosting with a blank page, because your code tells it to be blank

![image](https://user-images.githubusercontent.com/47092464/199969737-f8f15f48-bb1d-4e5e-9fce-8bc6f915ba30.png)

![image](https://user-images.githubusercontent.com/47092464/199969786-f7a34ae4-aad0-4495-a79a-b5077863a64c.png)

that's not an error just make sure you fix your code and by the way, the first time I opened that, it's asking for location and camera to be opened, which I turned down of course
